src/components/ListProducts/ListProducts.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
-import { useAxios } from '../../hooks/useAxios';
-import styles from './Products.module.css';
+// import { useAxios } from '../../hooks/useAxios';
+import styles from './ListProducts.module.css';
+import { Link } from 'react-router-dom';
+import { useGlobalContext } from '../../context/Provider';
 
-function Products() {
-    const { response, loading } = useAxios({
-        method: 'GET',
-        url: 'featured-products.json',
-    });
+function ListProducts({ fromHome }) {
+    const { featuredProduct, products, filtered } = useGlobalContext();
 
+    let toRender;
+    if (fromHome) {
+        toRender = featuredProduct;
+    } else {
+        toRender = filtered && filtered.length > 0 ? filtered : products;
+    }
```

Comment:
the ternary can be used instead of the `if\else` statement

```suggestion
    const toRender = fromHome
      ? featuredProduct
      : (filtered && filtered.length > 0) ? filtered : products;
```